openfisca_country_template/entities.py

```diff
@@ -1,13 +1,28 @@
 # -*- coding: utf-8 -*-
 
 # This file defines the entities needed by our legislation.
+import textwrap
 
 from openfisca_core.entities import build_entity
 
 Household = build_entity(
     key = "household",
     plural = "households",
     label = u'Household',
+    doc = textwrap.dedent('''
+    Household is an example of a group entity.
+    A group entity contains one or more natural persons. Each person in a group entity has a role (e.g. parent or children).
+    Some roles can only be held by a limited number of persons (e.g. a 'first_parent' can only be held by one person), where other can have unlimited number of persons (e.g. 'children').
+
+    Example:
+    Housing variables like 'housing_tax' are usually defined for a group entity such as 'Household'.
+
+    Usage:
+    Check the number of persons of a specific role (e.g. if there is a 'second_parent' with household.nb_persons(Household.SECOND_PARENT)).
```

Comment:
person --> individual
(e.g. check if there is a 'second_parent' with household.nb_persons(Household.SECOND_PARENT))